object-cache.php

```diff
@@ -479,7 +481,10 @@ function &get_mc( $group ) {
 	}
 
 	function failure_callback( $host, $port ) {
-		// error_log("Connection failure for $host:$port\n", 3, '/tmp/memcached.txt');
```

Comment:
It might get very noisy. Better to have some other handler watch for errors and then log and/or take appropriate action.